packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -526,9 +526,11 @@ export class DataStores implements IDisposable {
         // Iterate over each store and ask it to snapshot
         await Promise.all(Array.from(this.contexts)
             .filter(([_, context]) => {
-                // Summarizer works only with clients with no local changes!
+                // Summarizer works only with clients with no local changes. A data store in attaching
+                // state indicates an op was sent to attach a local data store.
                 assert(context.attachState !== AttachState.Attaching,
-                    0x165 /* "Summarizer cannot work if client has local changes" */);
+                    "Local data store detected in attaching state during summarize",
```

Comment:
curious, do we run process to tag asserts in release branches?

Reply:
I believe we do before releasing.